servers/Azure.Mcp.Server/README.md

```diff
@@ -118,40 +125,48 @@ npm install -g @azure/mcp
 
 ### Docker
 
-Microsoft publishes an official Azure MCP Server Docker container on the [Microsoft Artifact Registry](https://mcr.microsoft.com/artifact/mar/azure-sdk/azure-mcp).
+Pull the Docker image: [mcr.microsoft.com/azure-sdk/azure-mcp](https://mcr.microsoft.com/artifact/mar/azure-sdk/azure-mcp).
+
+```bash
+docker pull mcr.microsoft.com/azure-sdk/azure-mcp
+```
 
 <details>
-<summary>Docker setup instructions</summary>
+<summary>Docker instructions</summary>
+
+#### Create an env file with Azure credentials
 
 1. Create a `.env` file with Azure credentials ([see EnvironmentCredential options](https://learn.microsoft.com/dotnet/api/azure.identity.environmentcredential)):
 
 ```bash
-   AZURE_TENANT_ID={YOUR_AZURE_TENANT_ID}
-   AZURE_CLIENT_ID={YOUR_AZURE_CLIENT_ID}
-   AZURE_CLIENT_SECRET={YOUR_AZURE_CLIENT_SECRET}
+AZURE_TENANT_ID={YOUR_AZURE_TENANT_ID}
+AZURE_CLIENT_ID={YOUR_AZURE_CLIENT_ID}
+AZURE_CLIENT_SECRET={YOUR_AZURE_CLIENT_SECRET}
 ```
 
-2. Configure your MCP client with the Docker command:
+#### Configure your MCP client to use Docker:
+
+2. Run using the `.env` file from step 1. Replace `/full/path/to/your.env` with the actual `.env` file path.
 
 ```json
    {
-      "servers": {
+      "mcpServers": {
          "Azure MCP Server": {
             "command": "docker",
             "args": [
                "run",
                "-i",
                "--rm",
                "--env-file",
-               "/full/path/to/.env",
+               "/full/path/to/your.env",
                "mcr.microsoft.com/azure-sdk/azure-mcp:latest"
             ]
          }
       }
    }
 ```
-   
-Replace `/full/path/to/.env` with your actual .env file path. Alternatively, use individual `--env` flags or `--volume` mounts for credentials.
+**Note:** When manually configuring Visual Studio and Visual Studio Code, use `servers` instead of `mcpServers` as the root object.
+
```

Comment:
--volume is the use case where you want it to try and use your log in from existing connected apps like az cli.

Reply:
Please tell me exactly how you want the line changed :)

Reply:
It sounds like --volume is the recommended path and not an env file then?

Reply:
Added all the optional/note bullets back.  @conniey  will follow up next week to rewrite the section :)